Tasks/NuGetAuthenticateV1/taskJsonOverride.json

```diff
@@ -0,0 +1,41 @@
+{
+    "inputs": [
+        {
+            "name": "forceReinstallCredentialProvider",
```

Comment:
I think the order here matters for the task input user interface. Put service connection and feed URL at the top to be consistent with the other tasks.

Reply:
done